opentracing-noop/pom.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright 2016 The OpenTracing Authors
+
+    Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+    in compliance with the License. You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software distributed under the License
+    is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+    or implied. See the License for the specific language governing permissions and limitations under
+    the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.opentracing</groupId>
+        <artifactId>parent</artifactId>
+        <version>0.16-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>opentracing-noop</artifactId>
+    <name>OpenTracing-noop</name>
+    <description>OpenTracing NoOp</description>
+
+    <properties>
+        <main.basedir>${project.basedir}/..</main.basedir>
+    </properties>
```

Comment:
This is already defined and inherited from the top level pom.xml

![screenshot from 2016-10-03 15-46-25](https://cloud.githubusercontent.com/assets/559444/19027854/a8811a74-8980-11e6-847f-9351d67b9ee5.png)